### DISCUSSION TITLE
Auto-generate Copyrights?

### DISCUSSION THREAD

**Discussion by Teej42** [dotansimha/graphql-code-generator#5819] [Q&A] [answered]
Is there a way to tell the code generator to generate a pre-defined copyright notice for the generated code files?

**Comment by ardatan** [comment, marked as answer]
You can use "add" plugin;
https://www.graphql-code-generator.com/docs/plugins/add

**Reply by Teej42** [reply]
That appears to work well - The Add option does allow for newline (\n) to be used.

Unfortunately, I found out that the code generation is also done with mybatis too—more research to be done.

Thank you.
